content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,8 +6,6 @@ owner: istio/wg-networking-maintainers
 test: yes
 ---
 
-{{< boilerplate ambient-alpha-warning >}}
```

Comment:
I believe we can leave the Alpha warning on 1.21. It would be removed in master which reflects the 1.22 release.

Reply:
But if ambient is on beta why do we want to leave this warning?

Reply:
Yes, we should leave the warning. Sorry I missed that.

Reply:
understood, thanks for confirming I will add it shortly